src/assets/SASSAsset.js

```diff
@@ -35,6 +41,20 @@ class SASSAsset extends Asset {
       }
     });
 
+    opts.importer = (url, prev, done) => {
+      let resolved;
+      try {
+        resolved = syncPromise(
+          resolver.resolve(url, prev === 'stdin' ? this.name : prev)
+        ).path;
+      } catch (e) {
+        resolved = url;
```

Comment:
does this fall back to the default SASS resolver when Parcel doesn't find anything?